main/reference/zoe-api/zoe-data-types.md

```diff
@@ -63,14 +35,13 @@ You can get information about the contract instance via these methods:
 
 ## Invitation
 
-An **Invitation** is a kind of **[Payment](/reference/ertp-api/payment.md)**. It's a non-fungible eright
-that can be held in **Payments** or **[Purses](/reference/ertp-api/purse.md)**, just like any other
-eright. An **Invitation** **Payment** is a **Payment** holding an **Invitation**.
+An **Invitation** is a **[Payment](/reference/ertp-api/payment.md)** holding a non-fungible eright created by the **[InvitationIssuer](#invitationissuer)**.
+An **Invitation Payment** is a **Payment** holding an **Invitation**.
```

Comment:
These cannot both be true. In fact, the second sentence should be deleted. There is no distinct concept of "Invitation Payment" as distinct from "Invitation", since, as the first sentence says, an "Invitation" is already a kind of Payment.

Reply:
There is however a genuine terminology problem here. A purse or payment of the invitation issuers can have an amount with any number of invitation details, including none. Perhaps we need to switch to using "invitation" to describe the abstract non-fungible assets, which can move between purses and payments. In the same way that we wouldn't say that a dollar "is" a payment with a 1 dollar amount, we should probably stop saying that an invitation "is" a payment with 1 invitations detail amount.

What a mess. Would be a lot of work to clean up, but probably needed. Just do some adequate stopgap in this PR and file an issue? In any case, the above pair of sentences still need to be fixed because they contradict each other.

Reply:
Exclusively transferable assets cannot be reified as objects. Payments and Purses reify only containers for such assets, and amounts reify only descriptions of these assets. But the assets themselves only exist at a higher layer of abstraction. Ideally, "invitation" refers to the asset, not the payment holding the asset. But this contradicts a lot of current usage.

Hmm. Perhaps then the second sentence is more correct than the first, or should be going forward?

Reply:
I'm so glad my confusion here was warranted! Changed to this:
> An **Invitation** is a non-fungible asset created by the **[InvitationIssuer](#invitationissuer)**.
> An **Invitation Payment** is a **[Payment](/reference/ertp-api/payment.md)** holding an **Invitation**.

Reply:
Hm. I suggest escalating that to its own issue. There's a significant training burden around changing this sort of terminology. Getting everybody to stop saying "an invitation is a payment" is going to take some time/effort. We should perhaps have more discussion with devrel/product about how we want to talk about these things.

I suppose I don't object to changing this occurrence here/now. But let's not suppose that's all the work necessary to effect this change.

Reply:
#887

Reply:
With that issue filed, resolving here.